solvers/deep_mdd.py

```diff
@@ -25,7 +25,7 @@ class Solver(DeepDASolver):
     # the cross product for each key in the dictionary.
     # All parameters 'p' defined here are available as 'self.p'.
     default_param_grid = {
-        'criterion__reg': [1e-5, 1e-4, 1e-3, 1e-2, 1e-1, 1, 1e1, 1e2, 1e3],
+        'criterion__reg': [1e-2, 1e-1, 1],
     }
```

Comment:
For the `gamma` arg, you should define it in the param_grid like this:
```
'criterion__adapt_criterion': [
            MDDLoss(gamma=gamma, disc_criterion=torch.nn.CrossEntropyLoss())
            for gamma in [1, 2, 4]
        ],
```

Reply:
Oops 🙊

Reply:
Mais merci pour la review !